data/use_cases.yml

```
@@ -3,19 +3,34 @@
 developer_onboarding:
   title: Developer Onboarding
   link: /strategy-goals/strategy/use_cases/dev-onboarding
+  sponsors:
+    product: ben_venker
+    engineering: marek_zaluski
 
 code_reuse:
   title: Code Reuse
   link: /strategy-goals/strategy/use_cases/code_reuse
+  sponsors:
+    product: taylor_sperry
+    engineering: robert_lin
 
 code_health:
   title: Code Health
   link: /strategy-goals/strategy/use_cases/code_health
+  sponsors:
+    product: malo_marrec
+    engineering:
```

```suggestion
    engineering: joe_chen
```

I don't know how the `<first>_<last>` is determined at where?

---

It comes from your entry in the `team.yml`, which is usually just your name as you've written it here. In this case you've got it right. https://github.com/sourcegraph/handbook/blob/main/data/team.yml#L101

---

Gotcha, thank you!